code/modules/mob/living/carbon/human/species_types/ipc.dm

```diff
@@ -0,0 +1,40 @@
+/datum/species/ipc
+	// TOASTERS!
+	name = "IPC"
+	id = "ipc"
+	say_mod = "states"
+	heatmod = 2 // Went cheap with Aircooling
+	coldmod = 2 // Don't put your computer in the freezer.
+	burnmod = 1.5 // Overcoming augmented damage reduction - 1.5 puts it on par with unaugged organics.
+	brutemod = 1.5
+	species_traits = list(NOBREATH,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,EASYDISMEMBER,EASYLIMBATTACHMENT,NOPAIN,NO_BONES,NOCLONE,NOTRANSSTING,MUTCOLORS,TOXINLOVER) // NOHUNGER is temporary. A liver is needed to process chems (LIKE HOLY WATER)
```

Comment:
You'll probably need to code a special cybernetic liver for them to be able to process chems properly. Also, what's with `TOXINLOVER` being on in here?

Reply:
`TOXINLOVER` just makes them immune to toxins for the moment, because healing chems don't heal robotic limbs anyway.

I'll redo it when I add the substance processor.